syz-cluster/pkg/api/api.go

```diff
@@ -93,6 +86,38 @@ type Finding struct {
 	Log       []byte `json:"log"`
 }
 
+type Series struct {
+	ID          string        `json:"id"` // Only included in the reply.
+	ExtID       string        `json:"ext_id"`
+	Title       string        `json:"title"`
+	AuthorEmail string        `json:"author_email"`
+	Cc          []string      `json:"cc"`
+	Version     int           `json:"version"`
+	Link        string        `json:"link"`
+	PublishedAt time.Time     `json:"published_at"`
+	Patches     []SeriesPatch `json:"patches"`
+}
+
+func (s *Series) PatchBodies() [][]byte {
+	var ret [][]byte
+	for _, patch := range s.Patches {
+		ret = append(ret, patch.Body)
+	}
+	return ret
+}
+
+type SeriesPatch struct {
+	Seq   int    `json:"seq"`
+	Title string `json:"title"`
+	Link  string `json:"link"`
+	Body  []byte `json:"body"`
+}
+
+type NewSession struct {
+	ExtID string   `json:"ext_id"`
+	Tags  []string `json:"tags"`
+}
+
 // For now, there's no reason to obtain these really via a real API call.
```

Comment:
@melver told me about the `https://kernel.googlesource.com/pub/scm/linux/kernel/git/torvalds/linux`.
It may to be a better candidate for the defaultTrees.

Reply:
Thanks, I've added a TODO with the repository URL. We'll still need to double-check that it works fine before switching to it.